docs/streams.html

```diff
@@ -46,7 +46,7 @@
         <h2><a id="streams_overview" href="#streams_overview">Overview</a></h2>
 
         <p>
-        Kafka Streams is a client library for processing and analyzing data stored in Kafka and either write the resulting data back to Kafka or send the final output to an external system. It builds upon important stream processing concepts such as properly distinguishing between event time and processing time, windowing support, and simple yet efficient management of application state.
+        Kafka Streams is a client library for processing and analyzing data stored in Kafka and either write the resulting data back to Kafka, send the final output to an external system or expose its data using Interactive Queries. It builds upon important stream processing concepts such as properly distinguishing between event time and processing time, windowing support, and simple yet efficient management of application state.
```

Comment:
Not introduce in this PR, but can we break this into multiple lines? Best would be not longer than what Github shows to avoid scrolling :)

Also, please add `,`: `external system[,] or expose its`